vendor/bat-native-ads/src/bat/ads/internal/resources/frequency_capping/anti_targeting_resource.cc

```diff
@@ -18,8 +18,11 @@ namespace ads {
 namespace resource {
 
 namespace {
+
 const int kCurrentVersion = 1;
+const int kVersion = 1;
```

Comment:
What is the difference between `kCurrentVersion` and `kVersion` please. If we could rename `kVersion` to be more verbose that would be great

Reply:
as discussed yesterday - removed redundant version and added as feature parameter